src/lib/api/inline.preboot.code.ts

```diff
@@ -78,7 +79,10 @@ export const defaultOptions = <PrebootOptions>{
       preventDefault: true,
       freeze: true
     }
-  ]
+  ]).map(eventSelector => assign(eventSelector, {
+    // set `replay` to `true` if not specified
+    replay: eventSelector.replay != null ? eventSelector.replay : true
```

Comment:
I hesitate about doing this here, since it's a lot of work for those adding their own selectors to always add `replay`. Can you add this logic to the event replayer instead?

Something like `if(!!eventSelector.replay)` instead of `if(!eventSelector.noReplay)`

Reply:
This won’t be enough as undefinedis falsy. I’d have to compare to undefined explicitly. I’m a little afraid it might be fragile.

Reply:
Ah, I misread your comment. You meant it’s work for consumers, not Prevoot code. I agree then.